### PR TITLE
libperfmgr AIDL update

### PR DESCRIPTION
**Pull request by linckandrea**
DNM yet

### CODE REVIEW

File: sepolicy/vendor/file_contexts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (Flamefire) [inline sepolicy/vendor/file_contexts]: merge issue?

Comment (linckandrea) [reply sepolicy/vendor/file_contexts]: yes, thank you for pointing it out...  i pushed the wrong branch...

**Comment by linckandrea** [comment]
tested on my maple dsds device boot fine however i got some nodes reading errors

```
06-22 22:53:30.640   670   670 I android.hardware.power-service.yoshino-libperfmgr: Power HAL AIDL Service with Extension is starting.
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[0]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[0]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[1]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[1]'s DefaultIndex, set to last index: 4
06-22 22:53:30.641     0     0 I init    : Switched to default mount namespace
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[1]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[2]'s Type, set to 'File' as default
06-22 22:53:30.641     0     0 I init    : Not setting encryption policy on: /data/apex
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[2]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[3]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[3]'s DefaultIndex, set to last index: 5
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[3]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[4]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[4]'s DefaultIndex, set to last index: 3
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[4]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[5]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[5]'s DefaultIndex, set to last index: 2
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[5]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[6]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[6]'s DefaultIndex, set to last index: 2
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[6]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[7]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[7]'s DefaultIndex, set to last index: 1
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[7]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[8]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[8]'s DefaultIndex, set to last index: 1
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[8]'s ResetOnInit, set to 'false'
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[8]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[9]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[9]'s DefaultIndex, set to last index: 1
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[9]'s ResetOnInit, set to 'false'
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[9]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[10]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[10]'s DefaultIndex, set to last index: 1
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[10]'s ResetOnInit, set to 'false'
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[10]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[11]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[11]'s DefaultIndex, set to last index: 3
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[11]'s HoldFd, set to 'false'
06-22 22:53:30.641   670   670 E libperfmgr: Failed to read Node[12]'s Type, set to 'File' as default
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[12]'s DefaultIndex, set to last index: 1
06-22 22:53:30.641   670   670 I libperfmgr: Failed to read Node[12]'s ResetOnInit, set to 'false'
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[13]'s DefaultIndex, set to last index: 4
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[13]'s ResetOnInit, set to 'false'
06-22 22:53:30.642   668   668 D QSEECOMAPI: QSEECom_get_handle sb_length = 0xa000
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[14]'s DefaultIndex, set to last index: 1
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[14]'s ResetOnInit, set to 'false'
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[15]'s DefaultIndex, set to last index: 1
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[15]'s ResetOnInit, set to 'false'
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[16]'s DefaultIndex, set to last index: 1
06-22 22:53:30.642   670   670 I libperfmgr: Failed to read Node[16]'s ResetOnInit, set to 'false'
06-22 22:53:30.642   670   670 I libperfmgr: 17 Nodes parsed successfully
06-22 22:53:30.642   668   668 D QSEECOMAPI: App is already loaded QSEE and app id = 65537
06-22 22:53:30.642   670   670 I libperfmgr: 62 Actions parsed successfully
06-22 22:53:30.642   670   670 I libperfmgr: PowerHint AUDIO_LOW_LATENCY has 2 actions parsed
06-22 22:53:30.642   670   670 I libperfmgr: PowerHint AUDIO_STREAMING has 2 actions parsed
06-22 22:53:30.642   670   670 I libperfmgr: PowerHint CAMERA_LAUNCH has 4 actions parsed
06-22 22:53:30.642   670   670 I libperfmgr: PowerHint CAMERA_SHOT has 8 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint CAMERA_STREAMING has 5 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint EXPENSIVE_RENDERING has 3 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint INTERACTION has 7 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint LAUNCH has 8 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint SUSTAINED_PERFORMANCE has 4 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint VIDEO_ENCODE has 1 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint VR_MODE has 9 actions parsed
06-22 22:53:30.643   670   670 I libperfmgr: PowerHint VR_SUSTAINED_PERFORMANCE has 9 actions parsed
06-22 22:53:30.643     0     0 I init    : Not setting encryption policy on: /data/app-staging
06-22 22:53:30.643   670   670 I libperfmgr: Initialized HintManager from JSON config: /vendor/etc/powerhint.json
06-22 22:53:30.643     0     0 I init    : starting service 'apexd'...
06-22 22:53:30.643   670   670 I android.hardware.power-service.yoshino-libperfmgr: Initialize PowerHAL
06-22 22:53:30.643   670   670 I android.hardware.power-service.yoshino-libperfmgr: PowerHAL ready to process hints
```

**Comment by derfelot** [comment]
i've picked this stuff to 19 branches. those node reading errors are no problem afaict, we just don't have them.

**Pull request closed without merging**